pkg/migrations/op_create_table.go

```diff
@@ -336,6 +336,7 @@ type ConstraintSQLWriter struct {
 	Columns           []string
 	InitiallyDeferred bool
 	Deferrable        bool
+	SkipValidation    bool
```

Comment:
Can this be:
```suggestion
	NotValid    bool
```

so that it more closely maps to the SQL being generated?

Reply:
The name of the attribute in PostgreSQL is called `skip_validation` and in go `SkipValidation`: https://github.com/xataio/pg_query_go/blob/4fa82ad6d0369aab38c73903edf45287ddc31cd9/parser/include/postgres/nodes/parsenodes.h#L2684
Hence, the name. Do you still want me to change the name?

Reply:
Let's leave it then 😄